version/version.go

```diff
@@ -2,7 +2,7 @@ package version
 
 var (
 	// Version hold a semantic version of the running binary
-	Version = "0.0.1"
+	Version = "0.0.0"
```

Comment:
When run via Make the Version will be in format "v0.0.0"

Reply:
```suggestion
	Version = "v0.0.0"
```